zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/ElasticsearchStorageTest.java

```diff
@@ -160,66 +160,15 @@ public void check_unauthorized() {
       .isEqualTo("User: anonymous is not authorized to perform: es:ESHttpGet");
   }
 
-  @Test
```

Comment:
next step is moving the few tests we had around things like health check config up to the spring area.